wolfssl/wolfcrypt/types.h

```diff
@@ -2050,7 +2050,7 @@ enum Max_ASN {
                             /* Maximum DER digest ASN header size */
                             /* Max X509 header length indicates the
                              * max length + 2 ('\n', '\0') */
-    MAX_X509_HEADER_SZ  = (37 + 2), /* Maximum PEM Header/Footer Size */
+    MAX_X509_HEADER_SZ  = (48 + 2), /* Maximum PEM Header/Footer Size */
```

Comment:
This increases stack / memory usage for non PQ cases. Can you please add some processor macro logic to help optimize the `MAX_X509_HEADER_SZ` size?